server/graphql/document/src/mutations/encounter/update.rs

```diff
@@ -0,0 +1,74 @@
+use async_graphql::*;
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    document::encounter::{UpdateEncounter, UpdateEncounterError},
+};
+
+use crate::types::document::DocumentNode;
+
+#[derive(InputObject)]
+pub struct UpdateEncounterInput {
+    /// Encounter document data
+    pub data: serde_json::Value,
+    /// The schema id used for the counter data
+    pub schema_id: String,
+    pub parent: String,
```

Comment:
I think a comment for parent would be good

Reply:
done thanks